website/content/api-docs/operator/keyring.mdx

```diff
@@ -0,0 +1,156 @@
+---
+layout: api
+page_title: Keyring - Operator - HTTP API
+description: |-
+  The /operator/keyring endpoints provide tools for Nomad operators to interact with the root keyring.
+---
+
+# Keyring Operator HTTP API
+
+The `/operator/keyring` endpoints manage encryption root keys used for storing
+variables and signing workload, including examining active encryption keys,
+rotating keys, or removing unused keys.
+
+See the [Key Management] documentation for information how these capabilities
+are used. For a CLI to perform these operations manually, please see the
```

Comment:
```suggestion
are used. For instructions on how to use the CLI to perform these operations manually, please see the
```

This is a suggestion only. What you wrote it grammatically correct, it just didn't feel  natural to me.  This is just subjective feedback so feel free to ignore.

Reply:
Sure, that works